From aed7bf7ceec3d6373f2cf6bb812a1ec6d4c1568a Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Sun, 28 Jun 2020 01:54:42 +0200
Subject: [PATCH 084/100] DO NOT MERGE: drm/amdgpu: DC support for Kaveri and
 older

This reverts commit d9fda24.

NOTE: LVDS and VGA connector types are not supported
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_device.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
index 8a5a8ff5d362..4423e9e99532 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_device.c
@@ -3030,14 +3030,6 @@ bool amdgpu_device_asic_has_dc_support(enum amd_asic_type asic_type)
 	case CHIP_KAVERI:
 	case CHIP_KABINI:
 	case CHIP_MULLINS:
-		/*
-		 * We have systems in the wild with these ASICs that require
-		 * LVDS and VGA support which is not supported with DC.
-		 *
-		 * Fallback to the non-DC driver here by default so as not to
-		 * cause regressions.
-		 */
-		return amdgpu_dc > 0;
 	case CHIP_HAWAII:
 	case CHIP_CARRIZO:
 	case CHIP_STONEY:
-- 
2.17.1

